src/runtime/analytics-service-test.js

```diff
@@ -121,7 +144,12 @@ describes.realWin('AnalyticsService', {}, env => {
         const /* {?AnalyticsRequest} */ request =
           new AnalyticsRequest(firstArgument['buf']);
         expect(request.getEvent()).to.deep.equal(AnalyticsEvent.UNKNOWN);
-        analyticsService.logEvent(AnalyticsEvent.IMPRESSION_PAYWALL);
+        registeredCallback({
+          eventType: AnalyticsEvent.IMPRESSION_PAYWALL,
+          eventOriginator: EventOriginator.SWG_CLIENT,
+          isFromUserAction: null,
```

Comment:
maybe set userAction to true, set some parameters and check both event originator and user action on the other side?

Reply:
Analytics service isn't using the parameters at all right now and so I think testing them isn't necessary.  I did improve tests for originator and userAction